src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/SomaticGenotypingEngine.java

```diff
@@ -235,10 +235,13 @@ private void addGenotypes(final LikelihoodMatrix<Allele> tumorLog10Matrix,
                                         final Optional<LikelihoodMatrix<Allele>> normalLog10Matrix,
                                         final VariantContextBuilder callVcb) {
         final double[] tumorAlleleCounts = getEffectiveCounts(tumorLog10Matrix);
-        final Genotype tumorGenotype = new GenotypeBuilder(tumorSample, tumorLog10Matrix.alleles())
-                .AD(Arrays.stream(tumorAlleleCounts).mapToInt(x -> (int) FastMath.round(x)).toArray())
```

Comment:
Is `AD` still getting added to the genotype in the new code below?

Reply:
Yes, the original code wrote out AD based on the Bayesian estimate, then overwrote it during annotation because `DepthPerAlleleBySample` is a `StandardMutectAnnotation`.  So in theory, there didn't used to be a way to turn it off.

Reply:
I confirmed that the output without the arg does still have AD and added coverage in the tumor normal test.